source/common/network/transport_socket_options_impl.cc

```diff
@@ -77,9 +79,17 @@ TransportSocketOptionsUtility::fromFilterState(const StreamInfo::FilterState& fi
     needs_transport_socket_options = true;
   }
 
+  if (filter_state.hasData<ProxyProtocolFilterState>(ProxyProtocolFilterState::key())) {
+    const auto& proxy_protocol_filter_state =
```

Comment:
This part doesn't have test coverage:

https://storage.googleapis.com/envoy-pr/11584/coverage/source/common/network/transport_socket_options_impl.cc.gcov.html